File: android/src/main/java/com/getcapacitor/community/BitchatConfig.java
package com.getcapacitor.community;

import androidx.annotation.Nullable;

public class BitchatConfig {

    @Nullable
    Long announceInterval;

    public BitchatConfig(@Nullable Long announceInterval) {
        this.setAnnounceInterval(announceInterval);
    }

    public void setAnnounceInterval(@Nullable Long announceInterval) {
        this.announceInterval = announceInterval;
    }

    @Nullable
    public Long getAnnounceInterval() {
        return announceInterval;
    }
}

File: ios/Sources/BitchatPlugin/BitchatConfig.swift
import Capacitor

public class BitchatConfig {
    private var announceInterval: Int?

    init(announceInterval: Int?) {
        self.setAnnounceInterval(announceInterval)
    }

    func setAnnounceInterval(_ announceInterval: Int?) {
        self.announceInterval = announceInterval
    }

    func getAnnounceInterval() -> Int? {
        return self.announceInterval
    }
}
